datafusion/proto/proto/datafusion.proto

```diff
@@ -312,6 +312,10 @@ message LogicalExprNode {
     CubeNode cube = 23;
 
     RollupNode rollup = 24;
+
+    // boolean operations
+    IsTrue is_true = 25;
+    IsFalse is_false = 26;
```

Comment:
There is a mismatch between these names and the names used in the code. This change fixed the proto errors for me.

```suggestion
    IsTrue is_true_expr = 25;
    IsFalse is_false_expr = 26;
```

Reply:
Thanks!